subgraphs/smolverse/schema.graphql

```diff
@@ -27,12 +27,11 @@ type Collection @entity {
   "Internal for tracking attributes for a collection"
   _attributeIds: [String!]!
 
-  "Internal for tracking tokenIds minted for a collection"
-  _tokenIds: [String!]!
-
   name: String!
   standard: TokenStandard
   baseUri: String
+  tokensCount: BigInt!
+  stakedTokensCount: BigInt!
```

Comment:
For these counts, lets make them `Int!` since they will never reach the need for `BigInt!`.

Reply:
For sure, no idea why I used `BigInt`...